tests/compatibility-test.py

```diff
@@ -36,12 +42,15 @@ def setUpClass(cls):
         # from another local ray container. The local ray container
         # outside Kind environment has the same ray version as the
         # ray cluster running inside Kind environment.
-        utils.delete_cluster()
-        utils.create_cluster()
-        images = [ray_image, kuberay_operator_image, kuberay_apiserver_image]
-        utils.download_images(images)
-        utils.apply_kuberay_resources(images, kuberay_operator_image, kuberay_apiserver_image)
-        utils.create_kuberay_cluster(BasicRayTestCase.cluster_template_file,
+        K8S_CLUSTER_MANAGER.delete_kind_cluster()
+        K8S_CLUSTER_MANAGER.create_kind_cluster()
+        image_dict = {
+            CONST.RAY_IMAGE_KEY: ray_image,
+            CONST.OPERATOR_IMAGE_KEY: kuberay_operator_image
+        }
+        operator_manager = OperatorManager(image_dict)
+        operator_manager.prepare_operator()
```

Comment:
It looks like line 45 to 52 appears multiple times(BasicRayTestCase, RayFTTestCase, and RayServiceTestCase). Maybe we can combine them as a function?

Reply:
We decided not to combine them as a function after an offline discussion. Reasons: 

(1) Because defining a new function will couple both Kubernetes cluster and operator abstractions, it's hard to think of an easy-to-understand abstraction for the function consisting of L45 to L52. It will hurt the readability at this moment. 

(2) We may need to execute something between KinD cluster creation and operator creation. Take #750 as an example, create a new namespace with `restricted` Pod security standard.